### PR TITLE
388 landing page

### LINKED ISSUE #388: Build landing page
This should look roughly like this, and explain to our three types of users what kind of use they can make of the website:
Screen Shot 2021-01-29 at 5 42 56 PM

### PR DESCRIPTION
**Pull request by aleksssk**
Landing page

### CODE REVIEW

File: src/front-end/src/components/LandingPage/LandingPage.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+          </h1>
+          {/* Insert Image for Mentee here. */}
+          <center>
+            <h3>Mentee Image Place Holder</h3>
+          </center>
+          <p>
+            <font size="4">
+              <center>
+                You&apos;ll be matched with experienced individuals who can help
+                with the skills you&apos;ll need. Matches are made on common
```
Comment (jpabadir) [inline src/front-end/src/components/LandingPage/LandingPage.js]:
```suggestion
                with the skills you need. Matches are made on common
```

File: src/front-end/src/components/LandingPage/LandingPage.js [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            </center>
+          </h1>
+          {/* Insert Image for Mentor here. */}
+          <center>
+            <h3>Mentor Image Place Holder</h3>
+          </center>
+          <p>
+            <font size="4">
+              <center>
+                Accept mentees requests to teach them skills. Meetings are based
```
Comment (jpabadir) [inline src/front-end/src/components/LandingPage/LandingPage.js]:
```suggestion
                Accept mentees' requests to start teaching. Meetings are based
```

File: src/front-end/src/components/LandingPage/LandingPage.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+        </div>
+        <div id="menteeRight">
+          <h1>
+            <center>
+              <i> As a Mentee... </i>
+            </center>
+          </h1>
+          {/* Insert Image for Mentee here. */}
+          <center>
+            <h3>Mentee Image Place Holder</h3>
```
Comment (jpabadir) [inline src/front-end/src/components/LandingPage/LandingPage.js]:
```suggestion
            <h3>Mentee Image Placeholder</h3>
```

File: src/front-end/src/components/LandingPage/LandingPage.js [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+        </div>
+        <div id="mentorMiddle">
+          <h1>
+            <center>
+              <i> As a Mentor... </i>
+            </center>
+          </h1>
+          {/* Insert Image for Mentor here. */}
+          <center>
+            <h3>Mentor Image Place Holder</h3>
```
Comment (jpabadir) [inline src/front-end/src/components/LandingPage/LandingPage.js]:
```suggestion
            <h3>Mentor Image Placeholder</h3>
```

File: src/front-end/src/components/LandingPage/LandingPage.js [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+      <div className="container">
+        <div id="adminLeft">
+          <h1>
+            <center>
+              <i> As an Admin... </i>
+            </center>
+          </h1>
+          {/* Insert Image for Admin here. */}
+          <center>
+            <h3>Admin Image Place Holder</h3>
```
Comment (jpabadir) [inline src/front-end/src/components/LandingPage/LandingPage.js]:
```suggestion
            <h3>Admin Image Placeholder</h3>
```

File: src/front-end/src/components/LandingPage/LandingPage.js [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+          </center>
+          <p>
+            <font size="4">
+              <center>
+                Register your organization. Invite potential mentors from your
+                company. Embed the mentee form in your organization&apos;s
```
Comment (jpabadir) [inline src/front-end/src/components/LandingPage/LandingPage.js]:
```suggestion
                company. Embed the mentee signup form in your organization&apos;s
```

File: src/front-end/src/components/LandingPage/LandingPage.js [hunk 7]
```diff
@@ -0,0 +1,98 @@
...
+          {/* Insert Image for Mentor here. */}
+          <center>
+            <h3>Mentor Image Place Holder</h3>
+          </center>
+          <p>
+            <font size="4">
+              <center>
+                Accept mentees requests to teach them skills. Meetings are based
+                around your schedule. Choose as many or as little mentees to
+                mentor as desired.
```
Comment (jpabadir) [inline src/front-end/src/components/LandingPage/LandingPage.js]:
```suggestion
                around your schedule. Choose as many or as few mentees as you like.
```

**Review by jpabadir — APPROVED** [review]
Good PR! I added some minor suggestions you can easily commit from the "Files Changed" screen on GitHub's web interface. Please either accept them from there or discuss with me. Thanks!

**Comment by aleksssk** [comment]
Updated the comments JP wrote

**Pull request merged**